libblkid/src/superblocks/hfs.c

```diff
@@ -238,17 +242,23 @@ static int probe_hfsplus(blkid_probe pr, const struct blkid_idmag *mag)
 	    (memcmp(hfsplus->signature, "HX", 2) != 0))
 		return 1;
 
-	hfs_set_uuid(pr, hfsplus->finder_info.id, sizeof(hfsplus->finder_info.id));
-
+	/* Block size must be at least as large as a sector and a multiple of 2. */
```

Comment:
I didn't find docs for HFS that describe this field in detail, but other FS normally use power-of-two, not multiple-of-two.